<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogFindCircle</class>
 <widget class="QWidget" name="dialogFindCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuração de Calibração</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Raio mínimo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="tbRaioMin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>110</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Raio máximo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="tbRaioMax">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="value">
    <number>120</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btDefault">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Padrão</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTela">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>201</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbRaioMin">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbRaioMax">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>168</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
